src/main/java/frc/robot/subsystems/Carousel.java

```diff
@@ -44,12 +70,15 @@ public double getCurrent () {
     public void warmUp() {
         this.spin(Constants.CAROUSEL_SHOOTER_SPEED);
     }
-
-
+    
     public int getTicks() {
         return carouselEncoder.getRaw();
     }
 
+    public double getSlot() {
+        return -getTicks() / Constants.CAROUSEL_FIFTH_ROTATION_TICKS;
```

Comment:
Please add a comment about why there is a negative sign here.

Reply:
Good